conf/ocsci/early_testing_rhcos.yaml

```diff
@@ -2,6 +2,6 @@
 # This config will provide the necessary values for early testing of RHCOS based on RHEL 8.4
 ENV_DATA:
   early_testing: True
-  RELEASE_IMG: "registry.ci.openshift.org/rhcos-devel/rhel4784:4.7.0-rc.2"
+  RELEASE_IMG: "registry.ci.openshift.org/rhcos-devel/rhel4784:4.7.2"
   # vm_template_overwrite parameter overwrites the vm_template
-  vm_template_overwrite: "rhcos-47.84.202102221652-0-vmware.x86_64"
+  vm_template_overwrite: "rhcos-47.84.202103151537-0-vmware.x86_64"
```

Comment:
is it possible to create latest or something like that links to early testing?

Reply:
@vasukulkarni these images are build by Timothée Ravier as part of early testing RHOCS where all teams has to test. I am not sure whether they can tag the latest